packages/features/users/components/UserTable/UserListTable.tsx

```diff
@@ -117,18 +119,15 @@ export function UserListTable() {
   const [debouncedSearchTerm, setDebouncedSearchTerm] = useState("");
   const [isDownloading, setIsDownloading] = useState(false);
 
-  const [columnFilters, setColumnFilters] = useState<ColumnFiltersState>([]);
+  const columnFilters = useColumnFilters();
```

Comment:
We already have `?activeFilters=` on the search state (URL), but kept `columnFilters` separately with `onColumnFilterChange` below. I think it's unnecessary to keep two sources, so I've made `columnFilters` out of `useFiltersSearchState()`.